src/modules/database/database.service.ts

```diff
@@ -1,9 +1,16 @@
 import { Injectable, OnModuleInit } from '@nestjs/common';
 import { PrismaClient } from '@prisma/client';
+import { CreateUserDto } from '../users/dto/user.dto';
 
 @Injectable()
 export class DatabaseService extends PrismaClient implements OnModuleInit {
   async onModuleInit() {
     await this.$connect();
   }
+
+  async createUser(createUserDto: CreateUserDto) {
```

Comment:
This method should not be part of database service, it should be done on `users.service.ts`, importing the database service inside the service (also remove import of CreateUserDto once this change is made)